src/components/common/ClientIntakeForm/ClientIntakeForm.js

```diff
@@ -65,7 +65,6 @@ const ClientIntakeForm = onChange => {
             >
               CANCEL
             </Button>
-            {/* need to add functionality to both the cancel and submit button */}
```

Comment:
Always good to clean comments! Code looks good otherwise.

Reply:
this will be left in for future users of codebase as the submit is not fully connected at the moment